app/views/manage_outcomes/outcomes/_outcomes.html.erb

```diff
@@ -10,7 +10,7 @@
     <% course.outcomes.each do |outcome| %>
       <tr>
         <td><%= outcome %></td>
-        <td><%= outcome.assessments_count %></td>
+        <td><%= render "progress_bar", amount: outcome.active_assessments_count, total: outcome.active_assessments_count %></td>
```

Comment:
Working on that now. Wrapping it up before lunch hopefully.

On Tue, Dec 15, 2015 at 10:41 AM, Ali <notifications@github.com> wrote:

> In app/views/manage_outcomes/outcomes/_outcomes.html.erb
> <https://github.com/MIT-IR/abet/pull/126#discussion_r47652439>:
>
> > @@ -10,7 +10,7 @@
> >      <% course.outcomes.each do |outcome| %>
> >        <tr>
> >          <td><%= outcome %></td>
> > -        <td><%= outcome.assessments_count %></td>
> > +        <td><%= render "progress_bar", amount: outcome.active_assessments_count, total: outcome.active_assessments_count %></td>
>
> @derekprior <https://github.com/derekprior> I saw that you merged some of
> the changes we needed to get the progress bar to work. I wasn't sure how to
> get active assessments count with results or if that is still something you
> need to do.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MIT-IR/abet/pull/126/files#r47652439>.
>